pkg/render/fluentd_test.go

```diff
@@ -1051,3 +1086,44 @@ var _ = Describe("Tigera Secure Fluentd rendering tests", func() {
 		)
 	})
 })
+
+func setupEKSConfig() *render.EksCloudwatchLogConfig {
+	fetchInterval := int32(900)
+	return &render.EksCloudwatchLogConfig{
+		AwsId:         []byte("aws-id"),
+		AwsKey:        []byte("aws-key"),
+		AwsRegion:     "us-west-1",
+		GroupName:     "dummy-eks-cluster-cloudwatch-log-group",
+		FetchInterval: fetchInterval,
+	}
+}
+
+func setupInstallationConfig(t corev1.Toleration) *operatorv1.InstallationSpec {
```

Comment:
This is the type of function that I think we want to get rid of - it's wrapping one line of code.

There's really not much point in having a function that only wraps a single line of code - it just makes the calling code harder to read because it requires that you go read this function in order to understand it.

Reply:
It's much better to be verbose and duplicate lines of code than it is to be over-aggressive in commonizing into helper functions. I'd rather have readable code that takes a couple of more lines than minimized code that is fewer lines but harder to read.